data/anthony2021early/anthony2021early.yaml

```diff
@@ -0,0 +1,317 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Early induction of functional SARS-CoV-2-specific T cells associates with rapid
+  viral clearance and mild disease in COVID-19 patients
+doi: 10.1016/j.celrep.2021.108728
+description: >-
+  Measured SARS-CoV-2 viral load in the upper respiratory tract, along with SARS-CoV-2-specific
+  antibodies and T cell responses, at multiple time points from acute infection through
+  convalescence or until death.
+analytes:
+  swab_SARSCoV2_N:
+    description: >-
+      The swab_SARSCoV2_N metric refers to the detection of SARS-CoV-2 RNA in patient
+      swabs,using RT-PCR cycle counts as a proxy for viral quantity. Data for the
+      nasopharyngeal swab results were obtained from the combined dataset in the supplementary
+      materials of Challenger et al. (2022)
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    specimen: nasopharyngeal_swab
```

Comment:
Where did you find the information indicating the samples are nasopharyngeal swabs?

Reply:
I'm not sure if this specimen should be the same as other specimen in combined excel (nasopharyngeal_swab). Otherwise I think it should be unknown, because I think I can not find other one.